book/doctrine.rst

```diff
@@ -145,6 +141,54 @@ for you:
         collation-server     = utf8mb4_general_ci # Replaces utf8_general_ci
         character-set-server = utf8mb4            # Replaces utf8
```

Comment:
I think we should be changing our recommendation to use `utf8mb4` in all circumstances, just like we used to recommend `utf8` - its actually safer/better to use `utf8mb4` as it supports all utf8 chars.
With that in mind, I agree with @xabbuh that the purpose of this section is to help the user have the correct character set in all cases.